lib/mapbox/geocode.py

```diff
@@ -194,15 +208,14 @@ def reverse_geocode(latitude, longitude):
 
         results = response_json.get('features', [])
         if len(results) > 0:
-            result = results[0]
-            address = result['place_name']
+            info = extract_address_info(results)
```

Comment:
This breaks backwards compatibility for other code that is currently using this (unsure what); but in any case, if we are changing the shape of the return value, we should create a new function--unless we can change all callers.

I'd be curious to know what the immediate container with results is named.